synapse/handlers/e2e_keys.py

```diff
@@ -698,12 +699,12 @@ async def claim_one_time_keys(
         failures: Dict[str, JsonDict] = {}
 
         @trace
-        async def claim_client_keys(destination: str) -> None:
+        async def claim_client_keys(destination: str, user: UserID) -> None:
```

Comment:
I don't think you need to pass this in (like we don't pass in `remote_queries`)? It doesn't make much difference though.